src/Hydra/Action/DocumentationAction.php

```diff
@@ -30,15 +30,9 @@ public function __construct(ApiDocumentationBuilderInterface $apiDocumentationBu
 
     /**
      * Gets API doc.
-     *
-     * @param Request $request
-     *
-     * @return array
      */
-    public function __invoke(Request $request)
+    public function __invoke() : JsonResponse
     {
-        $request->attributes->set('_api_format', 'jsonld');
-
-        return $this->apiDocumentationBuilder->getApiDocumentation();
+        return new JsonResponse($this->apiDocumentationBuilder->getApiDocumentation(), JsonResponse::HTTP_OK, ['Content-Type' => 'application/ld+json']);
```

Comment:
It doesn't work anymore because the normalizer takes an object as input (not a raw array like here) and the request for this controller doesn't define `_api_resource_class` attribute.

IMO it's not a big deal because it doesn't make sense to return a Hydra doc encoded in another format than JSON-LD. But we can still change that later.